src/sql/src/func.rs

```diff
@@ -3838,7 +3838,7 @@ pub static MZ_CATALOG_BUILTINS: LazyLock<BTreeMap<&'static str, Func>> = LazyLoc
         "pretty_sql" => Scalar {
             params!(String, Int32) => BinaryFunc::PrettySql => String, oid::FUNC_PRETTY_SQL;
             params!(String) => Operation::unary(|_ecx, s| {
-                let width = HirScalarExpr::literal(Datum::Int32(100), ScalarType::Int32);
+                let width = HirScalarExpr::literal(Datum::Int32(mz_sql_pretty::DEFAULT_WIDTH.try_into().expect("must fit")), ScalarType::Int32);
```

Comment:
You could make the constant a 32-bit number, which should be plenty enough. Then you can do cheap up conversion and don't have to unwrap here.

Reply:
Good idea, will do!

Reply:
Actually, it's not so simple, because then the other use of `DEFAULT_WIDTH`, which gives it to `PrettyConfig::width`, would need to convert from i32 to usize. I could make `PrettyConfig::width` also an i32, but all these widths being signed types would look weird.

I think the original problem is that the `pretty_sql` scalar function takes a signed width. We could also change that, but changing the parameter types of an existing scalar function is probably more hassle than this is worth.

So, I'd like to just stay with these widths being unsigned types (as they should be), and just work around the problem that `pretty_sql` takes an `Int32` by doing a conversion there.